Kudu.Contracts/Diagnostics/DotNetCoreSharedFrameworksInfo.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Kudu.Services.Diagnostics
+{
+    public class DotNetCoreSharedFrameworksInfo
+    {
+        [JsonProperty(PropertyName = "Microsoft.NetCore.App")]
```

Comment:
nit: in #2890 this is specified as `NETCore` not `NetCore`

Reply:
Problem is that the serializer ignores the casing altogether and makes everything lower case. Do you know how to control this?

Reply:
I se, we globally set our `LowerCasePropertyNamesContractResolver()` which makes everything lowercase. Maybe best to keep things this way for consistency with the rest of the kudu API.